src/scenario/action_types/rome_wages.c

```diff
@@ -0,0 +1,47 @@
+#include "rome_wages.h"
+
+#include "city/data_private.h"
+#include "city/labor.h"
+#include "city/message.h"
+#include "core/random.h"
+
+void scenario_action_type_rome_wages_init(scenario_action_t *action)
+{
+    action->parameter4 = random_between_from_stdlib(action->parameter1, action->parameter2);
+}
+
+int scenario_action_type_rome_wages_execute(scenario_action_t *action)
+{
+    int32_t is_hard_set = action->parameter3;
+    int32_t adjustment = action->parameter4;
+    
+    city_data.labor.months_since_last_wage_change = 0;
+
+    int current_wages = city_labor_wages_rome();
+    if (is_hard_set
```

Comment:
Same here. For small multiple conditionals we don't use a newline. We only use it if the conditions breaks over the 120 char line limit.

Reply:
I'll remember that for the future and do a quick search to fix the current ones like this.
Done (with this one).